src/mame/pacman/pengo.cpp

```diff
@@ -821,13 +823,20 @@ void pengo_state::init_pengo6()
-GAME( 1982, pengo2,   pengo,    pengoe,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (set 2, encrypted)",            MACHINE_SUPPORTS_SAVE )
-GAME( 1982, pengo2u,  pengo,    pengou,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (set 2, rev A, not encrypted)", MACHINE_SUPPORTS_SAVE ) // Sega game ID# 834-5092 PENGO REV.A
-GAME( 1982, pengo3u,  pengo,    pengou,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (set 3, not encrypted)",        MACHINE_SUPPORTS_SAVE ) // Sega game ID# 834-5091 PENGO
-GAME( 1982, pengo4,   pengo,    pengoe,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (set 4, encrypted)",            MACHINE_SUPPORTS_SAVE ) // Sega game ID# 834-5081 PENGO
-GAME( 1982, pengo5,   pengo,    pengoe,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (set 5, encrypted)",            MACHINE_SUPPORTS_SAVE )
-GAME( 1982, pengo6,   pengo,    pengo,    pengo,    pengo_state, init_pengo6, ROT90, "Sega",                     "Pengo (set 6, encrypted)",            MACHINE_SUPPORTS_SAVE ) // Sega game ID# 834-5078 PENGO REV.A
-GAME( 1982, pengob,   pengo,    pengo,    pengo,    pengo_state, init_pengo6, ROT90, "bootleg",                  "Pengo (bootleg)",                     MACHINE_SUPPORTS_SAVE )
-GAME( 1982, penta,    pengo,    pengo,    pengo,    pengo_state, init_pengo6, ROT90, "bootleg (Grinbee Shouji)", "Penta",                               MACHINE_SUPPORTS_SAVE ) // Grinbee Shouji was a subsidiary of Orca
-GAME( 1983, jrpacmbl, jrpacman, jrpacmbl, jrpacmbl, pengo_state, empty_init,  ROT90, "bootleg",                  "Jr. Pac-Man (Pengo hardware)",        MACHINE_NO_COCKTAIL | MACHINE_SUPPORTS_SAVE )
+// World / Export releases do the pre-maze curtain effect instantly, draw the maze more quickly, have a short demo, and do not feature the 'popcorn' music.
+GAME( 1982, pengo,    0,        pengou,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (World, not encrypted, rev A)",    MACHINE_SUPPORTS_SAVE ) // Sega game ID# 834-5092 PENGO REV.A
+GAME( 1982, pengoa,   pengo,    pengoe,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (World, 315-5010 type) (set 1)",   MACHINE_SUPPORTS_SAVE )
+GAME( 1982, pengob,   pengo,    pengoe,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (World, 315-5010 type) (set 2)",   MACHINE_SUPPORTS_SAVE ) // for Bally?
+// this set is closest to the Japanese sets, with longer demo mode, and slower 'curtain draw' before the maze draw
+GAME( 1982, pengoc,   pengo,    pengoe,   pengo,    pengo_state, empty_init,  ROT90, "Sega",                     "Pengo (World, 315-5010 type) (set 3)",   MACHINE_SUPPORTS_SAVE ) // Sega game ID# 834-5081 PENGO
```

Comment:
Please just use one set of parentheses with all the descriptive text, e.g. “Pengo (World, 315-6010 type, set 1)”.